templates/documents.php

```diff
@@ -10,9 +10,7 @@
 </script>
 
 <?php
-style( 'richdocuments', 'share' );
```

Comment:
removed share.js as not needed @DeepDiver1975

Reply:
Meaning not used anywhere - was probably used by some legacy code back 2 years ago